ipwb/__main__.py

```diff
@@ -46,24 +46,31 @@ def checkArgs_replay(args):
 
         random.seed()
         # Write data to temp file (sub-optimal)
-        tempFilePath = tempfile.gettempdir() + '/' + ''.join(random.sample(
-              string.ascii_uppercase + string.digits * 6, 12)) + '.cdxj'
+
+        number_of_characters = 12
+        character_set = string.ascii_uppercase + string.digits * 6
+
+        tempFilePath = (
+            f"{tempfile.gettempdir()}{os.path.sep}"
+            f"{random.sample(character_set, number_of_characters)}.cdxj")
         with open(tempFilePath, 'w') as f:
             f.write(cdxjIn)
         args.index = tempFilePath
         suppliedIndexParameter = True
 
     proxy = None
     if hasattr(args, 'proxy') and args.proxy is not None:
-        print('Proxying to ' + args.proxy)
+        print(f'Proxying to {args.proxy}')
         proxy = args.proxy
 
     # TODO: add any other sub-arguments for replay here
     if suppliedIndexParameter:
         replay.start(cdxjFilePath=args.index, proxy=proxy)
     else:
         print('ERROR: An index file must be specified if not piping, e.g.,')
-        print('> ipwb replay /path/to/your/index.cdxj\n')
+        print((f"> ipwb replay "
+               f"{os.path.sep.join(['', 'path', 'to', 'your', 'index.cdxj'])}"
+               f"\n"))
```

Comment:
```suggestion
        print(("> ipwb replay "
               f"{os.path.sep.join(['', 'path', 'to', 'your', 'index.cdxj'])}"
               "\n"))
```

Reply:
I knew I was going to get the "unnecessary f-string" feedback with this, but the `f`s aligning seemed like a stylistic choice. :P

Reply:
(That I am willing to concede for the sake of progress, of course.)

Reply:
@machawk1 I think we can use `os.path.join` without the leading separator. Because Windows path does not start with `/` usually, instead a drive name is attached to it. So it would be better to say, `path\to\your\index.cdxj` instead of `\path\to\your\index.cdxj` ( and similar in the Unix style). After all, it is just an example path, so be it relative or absolute, does not matter.

I would change the second and third lines to something like:

```py
f"{os.path.join('path', 'to', 'your', 'index.cdxj')}\n"
```

A similar cases is somewhere else as well.

Reply:
Applied in 720c1c8